### PR TITLE
Add instructions for checking on allowed modules when using Appliance Webtasks

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/appliance/webtasks.md [hunk 1]
```diff
@@ -27,6 +34,70 @@ The `auth0-sandbox` is the recommended method for running your custom code.
...
+#### Running the `list_modules.js` Webtask
+
+To run the `List Modules` Webtask, which requires you to send an HTTP `GET` request, you must host the code to be executed so that the URL used returns the code. If you are in need of a place to host your code, consider using [GitHub Gist](https://gist.github.com/) or a similar service.
+
+For the purposes of this example, suppose that `http://example.com/webtask.js` leads to a JavaScript file containing your Webtask code.
+
+You would then execute the following command to `GET` a list of Webtask Modules available for your given Appliance instance:
+
+```
+curl https://webtask.<a0url>.com=http://example.com/webtask.js
```
Comment (tehsis) [inline articles/appliance/webtasks.md]: this needs to send an authorization token to it and also use the webtask_url paremeter.
I would recommend to go with wt-cli which is going to be easier to explain.

Instructions to use the CLI are in https://<manage_url>/#/account/webtasks

**Review by tehsis — CHANGES_REQUESTED** [review]

File: articles/appliance/webtasks.md [hunk 2]
```diff
@@ -27,6 +34,92 @@ The `auth0-sandbox` is the recommended method for running your custom code.
...
+        node_version: process.version,
+        modules: natives.concat(modules)
+    });
+};
+```
+
+Afterwards, create a webtask profile using `wt-cli`, if you don't already have one.
+
+```bash
+wt init --container "YOUR_TENANT_NAME" --url "https://sandbox.it.auth0.com" --token "eyJhbGci..." -p "a``YOUR_TENANT_NAME-default"
```
Comment (mpaktiti) [inline articles/appliance/webtasks.md]: @tehsis Not sure if I "translated" this right (I replaced `adcasdfa` with `YOUR_TENANT_NAME`).

Also, I was wondering whether `https://sandbox.it.auth0.com` stays the same for non-US customers. I've seen it as `https://sandbox-eu.it.auth0.com` for EU tenants but I'm not sure if it applies in this case.

File: articles/appliance/webtasks.md [hunk 3]
```diff
@@ -27,6 +34,92 @@ The `auth0-sandbox` is the recommended method for running your custom code.
...
+```
+
+::: panel-info wt init
+This is the very first command you will run once you install `wt-cli`. It is used to create and update webtask profiles. It also provisions a webtask token if you don't have one already. Each webtask profile has a name, a URL and a container. In order to configure a webtask profile you need to provide an email or phone number. You will receive a verification code there which you will have to input to the terminal. For more details you can use the wt built-in help: `wt init -h`.
```
Comment (mpaktiti) [inline articles/appliance/webtasks.md]: @tehsis Does this process (`You will receive a verification code...`) stays the same for appliance users or should I remove this?

File: articles/appliance/webtasks.md (hunk 2 above)
Comment (tehsis) [reply articles/appliance/webtasks.md]: That URL depends on the appliance installation, it should be WEBTASK_URL

File: articles/appliance/webtasks.md (hunk 3 above)
Comment (tehsis) [reply articles/appliance/webtasks.md]: No, appliances customers won't do this

Comment (tehsis) [reply articles/appliance/webtasks.md]: You still need to configure the profile (with a name, container and token) this is done by following the instructions on the webtask section on the account settings

**Review by tehsis — CHANGES_REQUESTED** [review]

**Pull request merged**